### PR TITLE
Full screen background 

### PR DESCRIPTION
**Pull request by TanyaStk**

### CODE REVIEW

File: Netflix/Screens/Onboarding/OnboardingViewController.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+//  OnboardingViewController.swift
+//  Netflix
+//
+//  Created by Tanya Samastroyenka on 25.05.2022.
+//
+
+import UIKit
+import SnapKit
+
+class OnboardingViewController: UIViewController {
```
Comment (dnv190) [inline Netflix/Screens/Onboarding/OnboardingViewController.swift]: Don't change anything, but usually it's better to stick to one style 
Here, you have view creation
1. as property, `let signInButton`
2. in the `ViewDidLoad` method

File: Podfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'Netflix' do
```
Comment (dnv190) [inline Podfile:4]: 👍

**Review by dnv190 — APPROVED** [review]

**Comment by TanyaStk** [comment]
https://user-images.githubusercontent.com/64659217/170675835-f5a3a919-2c11-4838-bed8-8a94c6f760e7.mov

File: Netflix/Screens/Onboarding/OnboardingViewController.swift [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+//
+//  Created by Tanya Samastroyenka on 25.05.2022.
+//
+
+import UIKit
+import SnapKit
+
+class OnboardingViewController: UIViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    
+    var pageController: UIPageViewController!
```
Comment (dnv190) [inline Netflix/Screens/Onboarding/OnboardingViewController.swift]:
```suggestion
    private lazy var pageController: UIPageViewController {
     pageController = UIPageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal, options: nil)
         pageController.dataSource = self
         pageController.delegate = self
         addChild(pageController)
         return pageController
    }()
```

File: Netflix/Screens/Onboarding/OnboardingViewController.swift [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        addChild(pageController)
+        view.addSubview(pageController.view)
+
+        let firstPageViewController = FirstPageViewController()
+        let secondPageViewController = SecondPageViewController()
+
+        controllers.append(firstPageViewController)
+        controllers.append(secondPageViewController)
+
+        pageController.setViewControllers([controllers[0]], direction: .forward, animated: false)
```
Comment (dnv190) [inline Netflix/Screens/Onboarding/OnboardingViewController.swift]: Please avoid using [0] - you can `get index out of range` exception
```suggestion
        if let firstViewController = controllers.first {
                pageController.setViewControllers([firstViewController], direction: .forward, animated: false)
        }
```
Or
`Array(t.prefix(1)` to get first element as an array

File: Netflix/Screens/Onboarding/OnboardingViewController.swift [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    func pageViewController(_ pageViewController: UIPageViewController, viewControllerBefore viewController: UIViewController) -> UIViewController? {
+        if let index = controllers.firstIndex(of: viewController) {
+            if index > 0 {
+                return controllers[index - 1]
+            } else {
+                return nil
+            }
+        }
+
+        return nil
```
Comment (dnv190) [inline Netflix/Screens/Onboarding/OnboardingViewController.swift]: imho, slightly easier to read :) 
```suggestion
guard 
      let index =  controllers.firstIndex(of: viewController)
      index > 0
else {
      return nil
 }
 return controllers[index - 1]
```

File: Netflix/Screens/Onboarding/OnboardingViewController.swift [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    func pageViewController(_ pageViewController: UIPageViewController, viewControllerAfter viewController: UIViewController) -> UIViewController? {
+        if let index = controllers.firstIndex(of: viewController) {
```
Comment (dnv190) [inline Netflix/Screens/Onboarding/OnboardingViewController.swift]: Please check the comment above

**Review by dnv190 — APPROVED** [review]
👍

**Pull request merged**